Modified Create a Github Account instructions

mikedao opened the pull request:
Changed instructions for Windows users to submit a key to Github.

`clip < ~/.ssh/id_rsa.pub` Does not work in windows, as `~/` is not a valid Windows path. Replaced it with `clip < %userprofile%\ssh\id_rsa.pub`, which is the correct way to reference the user directory path.

Also added a note stating that Windows Users cannot perform the SSH key verification process.  Windows does not come with an SSH client, and having a user install something like PuTTY just to verify a key.

This fixes the issues in the documentation now, but probably worth discussing is if Windows users should go through the trouble of setting up a SSH key at all or if they should just use HTTPS instead.

All tests pass, and the instructions have been checked locally.

tjgrathwell commented:
I tried this on my Windows 7 machine and the `clip < %userprofile%\ssh\id_rsa.pub` didn't work unless I quoted it, like `clip < "%userprofile%\ssh\id_rsa.pub"`.

![clip-fail](https://cloud.githubusercontent.com/assets/1041691/6456506/f4d1ab18-c119-11e4-8a85-db5a79623823.png)

Oddly it does work unquoted in a Windows 8.1 VM. Not sure what the difference is?

mikedao commented:
@tjgrathwell No idea. I'm going to chalk that one up to "It's feature enhancement customers get when upgrade to Windows 8! You don't have to use quotes anymore!"

It's a very silly thing to change so arbitrarily. How would you like to proceed? I can modify the documentation to cover both cases, or we can just make a note to use quotes with Windows 7 and not use quotes with Windows 8. Either approach is silly, but sadly necessary.

mikedao commented:
Updated the PR to have the instructions use quotes as discussed above. Windows 8 works with or without quotes, Windows 7 works only with quotes, so quotes it is to cover both cases. Squashed into one commit. Rake tests all pass, and visually checked to make sure there were no escape character weirdness taking place.

tjgrathwell commented:
Looks good! :+1:
